### PR TITLE
Add crashes since method

### PR DESCRIPTION
**Pull request by NickAb**

### CODE REVIEW

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/DeviceServer.kt [hunk 1]
```diff
@@ -36,6 +36,7 @@ import io.ktor.server.engine.ShutDownUrl
 import org.slf4j.LoggerFactory
 import java.io.File
 import java.net.NetworkInterface
+import java.time.Duration
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/DeviceServer.kt]: looks like redundant import

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/host/ISimulatorsNode.kt [hunk 2]
```diff
@@ -1,7 +1,9 @@
 package com.badoo.automation.deviceserver.host
 
 import com.badoo.automation.deviceserver.data.*
+import com.badoo.automation.deviceserver.ios.device.Device
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/host/ISimulatorsNode.kt]: redundant import ?

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/host/management/IDeviceManager.kt [hunk 3]
```diff
@@ -1,7 +1,9 @@
 package com.badoo.automation.deviceserver.host.management
 
 import com.badoo.automation.deviceserver.data.*
+import com.badoo.automation.deviceserver.ios.device.Device
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/host/management/IDeviceManager.kt]: redundant import?

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt [hunk 4]
```diff
@@ -590,6 +590,22 @@ class Simulator (
...
+    override fun crashLogs(sinceMinutesAgo: Duration?): List<CrashLog> {
+        var crashLogFiles = listCrashLogs(sinceMinutesAgo)
+
+        var crashLogs = crashLogFiles.map {
+            val rv = remote.execIgnoringErrors(listOf("cat", it))
+
+            if (rv.isSuccess) {
+                CrashLog(filename = File(it).name, content = rv.stdOut)
+            } else {
+                null
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt:588]: maybe log reason if not successful ?

**Review by fr0l — CHANGES_REQUESTED** [review]

**Comment by NickAb** [comment]
@fr0l, review please

**Pull request merged**